### PR TITLE
Adding functionality to ElasticSearch module

### PR DESCRIPTION
**Pull request by Avilir**
Adding functions to open connection, getting list of all 'tables' in the ES and copy all data to different ES server

Signed-off-by: Avi Liani <alayani@redhat.com>

### CODE REVIEW

File: ocs_ci/ocs/elasticsearch.py [hunk 1]
```diff
@@ -214,3 +219,56 @@ def local_server(self):
         proc = subprocess.Popen(cmd, shell=True)
         log.info(f'Starting LocalServer with PID of {proc.pid}')
         self.lspid = proc.pid
+
+    def _es_connect(self):
+        """
+        Create a connection to the ES via the localhost port-fwd
+
+        Returns:
+            obj: elasticsearch connection object
```
Comment (ebenahar) [inline ocs_ci/ocs/elasticsearch.py:228]: `Elasticsearch: elasticsearch connection object`

Comment (Avilir) [reply ocs_ci/ocs/elasticsearch.py]: Changed.

File: ocs_ci/ocs/elasticsearch.py [hunk 2]
```diff
@@ -12,6 +12,8 @@
 import subprocess
 import time
 
+from elasticsearch import (Elasticsearch, exceptions as esexp)
```
Comment (ebenahar) [inline ocs_ci/ocs/elasticsearch.py:15]: `()` are redundant

File: ocs_ci/ocs/elasticsearch.py [hunk 3]
```diff
@@ -214,3 +219,56 @@ def local_server(self):
         proc = subprocess.Popen(cmd, shell=True)
         log.info(f'Starting LocalServer with PID of {proc.pid}')
         self.lspid = proc.pid
+
+    def _es_connect(self):
+        """
+        Create a connection to the ES via the localhost port-fwd
+
+        Returns:
+            Elasticsearch: elasticsearch connection object
```
Comment (ebenahar) [inline ocs_ci/ocs/elasticsearch.py:228]: please add `Raises` to the docstring

File: ocs_ci/ocs/elasticsearch.py [hunk 4]
```diff
@@ -214,3 +219,56 @@ def local_server(self):
...
+        try:
+            es = Elasticsearch([{'host': 'localhost', 'port': self.get_port()}])
+        except esexp.ConnectionError:
+            log.error('Can not connect to ES server in the LocalServer')
+            raise
+        return es
+
+    def get_indices(self):
+        """
+        Getting list of all indexes in the ES server - all created by the test,
```
Comment (ebenahar) [inline ocs_ci/ocs/elasticsearch.py:239]: Indices

File: ocs_ci/ocs/elasticsearch.py [hunk 5]
```diff
@@ -214,3 +219,56 @@ def local_server(self):
...
+        the installation of the ES was without any indexes pre-installed.
+
+        Returns:
+            list : list of all indexes defined in the ES server
```
Comment (ebenahar) [inline ocs_ci/ocs/elasticsearch.py:243]: Indices

File: ocs_ci/ocs/elasticsearch.py [hunk 6]
```diff
@@ -214,3 +219,56 @@ def local_server(self):
...
+
+        """
+        results = []
+        log.info("Getting all indexes")
```
Comment (ebenahar) [inline ocs_ci/ocs/elasticsearch.py:247]: Indices

File: ocs_ci/ocs/elasticsearch.py (hunk 2 above)
Comment (Avilir) [reply ocs_ci/ocs/elasticsearch.py]: done

File: ocs_ci/ocs/elasticsearch.py (hunk 3 above)
Comment (Avilir) [reply ocs_ci/ocs/elasticsearch.py:228]: done

File: ocs_ci/ocs/elasticsearch.py (hunk 4 above)
Comment (Avilir) [reply ocs_ci/ocs/elasticsearch.py]: done

File: ocs_ci/ocs/elasticsearch.py (hunk 5 above)
Comment (Avilir) [reply ocs_ci/ocs/elasticsearch.py]: done

File: ocs_ci/ocs/elasticsearch.py (hunk 6 above)
Comment (Avilir) [reply ocs_ci/ocs/elasticsearch.py]: done

**Comment by vasukulkarni** [comment]
If its verified, Please add 'verified' label.

**Pull request merged**